public/app/plugins/panel/canvas/components/connections/ConnectionAnchors.tsx

```diff
@@ -114,36 +114,35 @@ export const ConnectionAnchors = ({ setRef, handleMouseLeave }: Props) => {
 };
 
 const getStyles = (theme: GrafanaTheme2) => ({
-  root: css`
-    position: absolute;
-    display: none;
-  `,
-  mouseoutDiv: css`
-    position: absolute;
-    margin: -30px;
-    width: calc(100% + 60px);
-    height: calc(100% + 60px);
-  `,
-  anchor: css`
-    padding: ${ANCHOR_PADDING}px;
-    position: absolute;
-    cursor: cursor;
-    width: calc(5px + 2 * ${ANCHOR_PADDING}px);
-    height: calc(5px + 2 * ${ANCHOR_PADDING}px);
-    z-index: 100;
-    pointer-events: auto !important;
-  `,
-  highlightElement: css`
-    background-color: #00ff00;
-    opacity: 0.3;
-    position: absolute;
-    cursor: cursor;
-    position: absolute;
-    pointer-events: auto;
-    width: 16px;
-    height: 16px;
-    border-radius: ${theme.shape.radius.circle};
-    display: none;
-    z-index: 110;
-  `,
+  root: css({
+    position: 'absolute',
+    display: 'none',
+  }),
+  mouseoutDiv: css({
+    position: 'absolute',
+    margin: '-30px',
+    width: 'calc(100% + 60px)',
+    height: 'calc(100% + 60px)',
+  }),
+  anchor: css({
+    padding: `${ANCHOR_PADDING}px`,
+    position: 'absolute',
+    cursor: 'cursor',
+    width: `calc(5px + 2 * ${ANCHOR_PADDING}px)`,
+    height: `calc(5px + 2 * ${ANCHOR_PADDING}px)`,
+    zIndex: 100,
+    pointerEvents: 'auto',
```

Comment:
losing the `!important` here didn't seem to impact the functionality of connection anchors 🤷‍♂️  side note I'm not sure it's possible to include `!important` notation via the css object notation

Reply:
I've seen it being used in code, but it didn't work for me.. so not sure if it actually works 😬